apollo-router/src/axum_factory/tests.rs

```diff
@@ -2320,7 +2320,6 @@ async fn test_supergraph_timeout() {
         QueryAnalysisLayer::new(supergraph_creator.schema(), Arc::clone(&conf)).await,
         Arc::new(supergraph_creator),
         conf.clone(),
-        Default::default(),
```

Comment:
Super pleased that the manifest poller is no-longer leaking out of the layer.

Reply:
Agreed, though there is a downside there — a pattern for (correctly!) preserving state over config reloads would be interesting.